ices/67848.rs

```diff
@@ -0,0 +1,9 @@
+trait WithAssoc { type AssocType; }
```

Comment:
Looks like there's a typo in this filename, rust-lang/rust#67848 is a PR

Reply:
Good point! It should be 67844.